src/Aop/Support/AnnotatedReflectionProperty.php

```diff
@@ -0,0 +1,60 @@
+<?php
+declare(strict_types = 1);
+/*
+ * Go! AOP framework
+ *
+ * @copyright Copyright 2014, Lisachenko Alexander <lisachenko.it@gmail.com>
```

Comment:
Year isn't correct :)

Reply:
I did copy/paste of docBlock.

Different classes have different years, I checked.

To be honest - I am ok for having author and/or copyright annotation, but year in docblock does not have any particular value, we could just remove it - it poses only unnecessary maintenance.

Reply:
> To be honest - I am ok for having author and/or copyright annotation, but year in docblock does not have any particular value, we could just remove it - it poses only unnecessary maintenance.

Looks reasonable. Accepted, will do this for the master branch.